source/MaterialXTest/MaterialXCore/Node.cpp

```diff
@@ -164,6 +164,7 @@ TEST_CASE("Flatten", "[nodegraph]")
 
     // Flatten all root-level nodes.
     doc->flattenSubgraphs();
+    doc->validate();
```

Comment:
Wouldn't we need to check the return value of this validate call with a `REQUIRE`?